products/quasar.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Quasar
+category: framework
+tags: javascript-runtime
+iconSlug: quasar
+permalink: /quasar
+versionCommand: quasar -v
+releasePolicyLink: https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule
+changelogTemplate: https://github.com/quasarframework/quasar/releases/tag/quasar-v__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+-   npm: quasar
+
+identifiers:
+-   purl: pkg:npm/quasar
+
+releases:
+-   releaseCycle: "2"
+    support: true
+    eol: false
+    latest: "2.12.7"
+    lts: false
+    latestReleaseDate: 2023-09-13
+    releaseDate: 2021-07-21
```

Comment:
```suggestion
    releaseDate: 2021-06-21
```

According to https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md#support-policy-and-schedule.